lib/assets/stylesheets/modules/_warning-text.scss

```diff
@@ -0,0 +1,73 @@
+.govuk-warning-text {
+  font-family: nta,Arial,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: #0b0c0c;
```

Comment:
we should probably be using colours from the [palette](https://github.com/alphagov/tech-docs-gem/blob/b650d313b893af4990603c1cff917ab1776a0d65/lib/assets/stylesheets/govuk_frontend_toolkit/colours/_palette.scss) and other stuff.